src/viper/syntax.ml

```diff
@@ -11,10 +11,13 @@ type prog = (item list, info) Source.annotated_phrase
 and item = (item', info) Source.annotated_phrase
 and item' =
   (* | import path *)
+  | AdtI of string * string list * adt_con list
```

Comment:
```suggestion
  | AdtI of id * id list * adt_con list
```
I think it should be string with location of original ids from the source code.
I know that we are not checking this in tests and even manually, but I think it's a good thing to keep common style of definitions at least in types. In the translation you can put `Source.no_region` as a stab value.